move-vm-backend-common/src/gas_schedule.rs

```diff
@@ -28,122 +28,122 @@ lazy_static! {
     /// A predefined gas strategy for instruction table cost.
```

Comment:
Content at line 28 `// TODO(rqnsom): tweak the cost for intructions` can be removed now

Reply:
Done :+1: